content/en/docs/concepts/observability-primer.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Observability Primer
+weight: 9
+---
+
+Before digging into OpenTelemetry, it is important to understand some core concepts first.
+
+## What is Observability?
+
+Observability lets us understand a system from the outside, by letting us ask questions about that system without knowing its inner workings. Furthermore, allows us to easily troubleshoot and handle novel problems (i.e. “unknown unknowns”), and helps us answer the question, "Why is this happening?" *without changing your code*.
+
+In order to be able to ask those questions of a system, code must be properly instrumented. **OpenTelemetry** is the mechanism by which application code is instrumented, to help make a system observable.
```

Comment:
Is an explanation of "instrumented" required? Can we point to the glossary?

Reply:
I elaborated in the paragraph, as per above.